slackbot/slackclient.py

```diff
@@ -130,6 +130,12 @@ def send_message(self, channel, message, attachments=None, as_user=True):
     def get_channel(self, channel_id):
         return Channel(self, self.channels[channel_id])
 
+    def get_user(self, userid):
+        user = self.webapi.users.info(userid)
+        if not user.successful:
+            raise RuntimeError(user.error)
+        return user.body["user"]
```

Comment:
That's a good idea—I'll see if I can improve it. By the way, any suggestions for how to write a test case for this? I haven't completely understood how you've set up the test environment.